### ISSUE TITLE
Theme Progress

### ISSUE THREAD

**Issue by Sand216** [SandTechStuff/10ThemeFor11#1]
Outdated with Vibranium11 release.

**Comment by ErikaXGH** [comment]
add action center animation please

**Comment by Sand216** [comment]
@ErikaXGH That cannot be restored solely through msstyles.

**Comment by ErikaXGH** [comment]
> @ErikaXGH That cannot be restored solely through msstyles.

What is the file for the action center animation?

**Comment by Sand216** [comment]
@ErikaXGH that is out of the scope of this project.

**Comment by Hunyar08** [comment]
Could you please make one for Windows 11 21H2? I know it's basically the same, but I need it for a specific project, and since it's Windows 11 it could be significantly easier to recreate, from my assumption, at least.

**Comment by Sand216** [comment]
@Hunyar08 Does it not work properly on 21H2? If so, what's wrong?

**Comment by Hunyar08** [comment]
@SandTechStuff When you swap out the Themes folder in Windows 11 24H2 for the one from 21H2, you get the same problems as when you swap out the folder in Windows 10. Right-click menus stop working, Explorer won't open, etc.

**Comment by Sand216** [comment]
I’m not entirely sure what you mean. Are you trying to use 10ThemeFor11 on 21H2? Are you talking about a port of 21H2’s theme to 24H2?

**Comment by Hunyar08** [comment]
@SandTechStuff Sorry for the poor wording. Yes, I was talking about a port of 21H2's theme to 24H2. I was saying that it could be easier since Windows 11 21H2 after all is Windows 11 and Windows 11 24H2 is of course Windows 11 too.

**Comment by Hunyar08** [comment]
@SandTechStuff Do you think you could make it?

**Comment by Hunyar08** [comment]
sand? you there?

**Comment by Sand216** [comment]
Yes I could, afaik there are only 2 differences from 24H2, but it's not a priority.

**Comment by Hunyar08** [comment]
can you tell me the differences? maybe just only send me it?

**Comment by Hunyar08** [comment]
@SandTechStuff  Please dude. I'm begging. JUST make a port for it on 24h2. its only two differences so it shouldnt be that hard anyway. if you cant, send only me it through my buisness email: mamakal2011@gmail.com or tell me how to change it so i can use it. PLEASE RESPOND PLEASE

**Comment by Sand216** [comment]
> can you tell me the differences?

Oh, I thought you would know them already. Afaik the button pressed image, dark mode maximize/restore buttons, and context menus are different. This is if you are talking about Windows 11 21h2, if you are talking about Windows 10 21h2 I don't think there are any differences.

> if you cant, send only me it through my buisness email

I don't see a reason to, I'd just send it here so others could use it.

Any particular reason for wanting this? If you want it ASAP than you might be better off doing it yourself with [msstyleEditor](https://github.com/nptr/msstyleEditor) as I have other things to do. If you are interesting, you can get a copy of the 24H2 and 21H2 msstyles and export the images that are different in 21H2 into the 24H2 msstyles.

**Comment by aubymori** [comment]
> [@SandTechStuff](https://github.com/SandTechStuff) Please dude. I'm begging. JUST make a port for it on 24h2. its only two differences so it shouldnt be that hard anyway. if you cant, send only me it through my buisness email: [mamakal2011@gmail.com](mailto:mamakal2011@gmail.com) or tell me how to change it so i can use it. PLEASE RESPOND PLEASE

Dude, if you're that desperate, just do it yourself.

**Comment by Hunyar08** [comment]
I found out how to do it. Don't need yall anymore :)

**Comment by Hunyar08** [comment]
I need help with changing them. what are the classes and parts that I need to change in order to get it to work? If you can, can you send an image or tell me which ones I need to replace? I've been trying. But I haven't been able to get it so that when you highlight over an item it turns blue. Its still gray like the default, even when changing the accent color to blue. Help.

**Comment by Hunyar08** [comment]
I MADE IT!!!!! ty

**Comment by whiskier07** [comment]
Hello, I am curious what needs to be modified in order to only keep the 10themefor11 animations while the rest of the default windows 11 theme remains untouched?

It conflicts with the dark theme used by StartAllBack

**Comment by Louis047** [comment]
@Hunyar08 hey there, mind sending the theme port for windows 11 24h2 ? i'm currently in 24H2 as well.

**Comment by Louis047** [comment]
@SandTechStuff is there any further progress for this ? i see that 99% of the work is done except for the last one and its been almost a year already. 

I'm currently in 24H2, will this work for me or does it break windows completely ?

**Comment by Sand216** [comment]
@Louis047 currently remaking this theme from the ground up. The current version should still work on updated 24h2 and 25h2 however.

**Comment by Louis047** [comment]
@Sand216 hey, glad to hear this! would it take longer or sooner ? Can't wait to try it out once released :)

And yes as we discussed in discord earlier, it does work in 24h2 but theres an issue with it currently as it doesn't properly support the new aero theme due to it missing few classes that were added to 24h2+ builds. Ive already posted about this weeks back #7
